app/model.js

```diff
@@ -22,4 +22,28 @@ exports.selectAllArticles = ()=>{
 
 exports.selectSpecifiedComments = (id)=>{
     return db.query(`SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC`,[id])
-}
\ No newline at end of file
+}
+
+exports.updateComments = (comment,id)=>{
+
+    const {body} = comment
+    const {article_id} = id
+    const author = comment.username
+ 
+    if(body===undefined||article_id===undefined||author===undefined){
+        console.log("hello")
+       return err
+    }
+
+    return db.query(`SELECT username FROM users WHERE name =$1`,[author]).then(({rows})=>{
```

Comment:
you shouldn't be querying the database for a name here, the posted comment should just be a valid username in the database, no need to use names